speaker/main.go

```diff
@@ -71,7 +71,6 @@ func main() {
 		host       = flag.String("host", os.Getenv("METALLB_HOST"), "HTTP host address")
 		mlBindAddr = flag.String("ml-bindaddr", os.Getenv("METALLB_ML_BIND_ADDR"), "Bind addr for MemberList (fast dead node detection)")
 		mlBindPort = flag.String("ml-bindport", os.Getenv("METALLB_ML_BIND_PORT"), "Bind port for MemberList (fast dead node detection)")
-		mlLabels   = flag.String("ml-labels", os.Getenv("METALLB_ML_LABELS"), "Labels to match the speakers (for MemberList / fast dead node detection)")
```

Comment:
Needs a mention in release notes, so people with custom manifests know to update, or they'll end up with a crashloop for unrecognized flags.

Reply:
done